std/bigint.d

```diff
@@ -578,7 +582,13 @@ string toHex(const(BigInt) x)
     return outbuff;
 }
 
-// Returns the absolute value of x converted to the corresponding unsigned type
+/** Returns the absolute value of x converted to the corresponding unsigned
```

Comment:
Probably better to use an actual `Returns:` section so that return value descriptions get a consistent, standard formatting.